treeherder/etl/management/commands/read_pulse_pushes.py

```diff
@@ -3,30 +3,30 @@
 from kombu import (Connection,
                    Exchange)
 
-from treeherder.etl.pulse_consumer import ResultsetConsumer
+from treeherder.etl.pulse_consumer import PushConsumer
 
 
 class Command(BaseCommand):
 
     """
-    Management command to read resultsets from a set of pulse exchanges
+    Management command to read pushes from a set of pulse exchanges
 
-    This adds the resultsets to a celery queue called ``store_pulse_resultsets`` which
-    does the actual storing of the resultsets in the database.
+    This adds the pushes to a celery queue called ``store_pulse_resultsets`` which
+    does the actual storing of the pushes in the database.
     """
 
-    help = "Read resultsets from a set of pulse exchanges and queue for ingestion"
+    help = "Read pushes from a set of pulse exchanges and queue for ingestion"
 
     def handle(self, *args, **options):
         config = settings.PULSE_DATA_INGESTION_CONFIG
         assert config, "PULSE_DATA_INGESTION_CONFIG must be set"
-        sources = settings.PULSE_RESULTSET_SOURCES
-        assert sources, "PULSE_RESULTSET_SOURCES must be set"
+        sources = settings.PULSE_PUSH_SOURCES
```

Comment:
Please add a comment to the PR stating that these env vars will need to be set to the new names on "prototype", "stage" and "production"  prior to merging to master.